sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/Mode.scala

```diff
@@ -70,10 +76,30 @@ case class Mode(
     buffer
   }
 
-  override def eval(buffer: OpenHashMap[AnyRef, Long]): Any = {
-    if (buffer.isEmpty) {
+  override def eval(buff: OpenHashMap[AnyRef, Long]): Any = {
```

Comment:
Could we avoid the change ?

Reply:
Either I am renaming here or line 83. If I change line 83 instead, I will have to change all of the references to buffer on L85 and L86. Do you have a particular preference and if so, why?

Reply:
Some changes may be unavoidable, but since we can choose - let's opt for keeping the method signature intact